packages/mdc-textfield/index.js

```diff
@@ -75,7 +78,37 @@ export class MDCTextfield extends MDCComponent {
           label.classList.remove(className);
         }
       },
-    }, this.getInputAdapterMethods_(), this.getHelptextAdapterMethods_()));
+    }, this.getInputAdapterMethods_(), this.getHelptextAdapterMethods_(), this.getBottomLineAdapterMethods_()));
+  }
+
+  getBottomLineAdapterMethods_() {
+    return {
+      addClassToBottomLine: (className) => {
+        if (this.bottomLine_) {
```

Comment:
hmmm, I do find it odd that a method could be a no-op....but its keeping with a precedent so lets leave it for now.